source/brailleTables.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding:utf-8 -*-

```

Comment:
This encoding declaration is invalid. This should be:
```suggestion
# -*- coding:utf-8 -*-
# -*- coding: UTF-8 -*-
```

Reply:
@cary-rowen did you see this comment?

Reply:
@cary-rowen did you see this comment?